usage/conf/usage.conf

```diff
@@ -0,0 +1,28 @@
+env USER=media-service
+env USER_HOME=/home/media-service
+env JAR=/home/media-service/app.jar
+
+# Workaround for https://github.com/aws/aws-sdk-java/issues/123
+env JAVA_OPTS="-XX:MaxRAMFraction=2 -XX:InitialRAMFraction=2 -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:gc.log"
```

Comment:
I think they should all use the ratio method here TBH.

On Thu, 22 Oct 2015 at 13:55 Sébastien Cevey <notifications@github.com>
wrote:

> In usage/conf/usage.conf
> <https://github.com/guardian/grid/pull/1301#discussion_r42743845>:
>
> > @@ -0,0 +1,28 @@
> > +env USER=media-service
> > +env USER_HOME=/home/media-service
> > +env JAR=/home/media-service/app.jar
> > +
> > +# Workaround for https://github.com/aws/aws-sdk-java/issues/123
> > +env JAVA_OPTS="-XX:MaxRAMFraction=2 -XX:InitialRAMFraction=2 -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:gc.log"
>
> I'd probably supersticiously set it, yeah. re memory, whichever is best I
> don't mind, but might be good to have consistency across the different run
> scripts, or else document why they differ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/grid/pull/1301/files#r42743845>.
>

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396